lib/helpUIOpener.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+// const interact = require('./interactWStatusBox.js');
```

Comment:
remove this commented out code if it was just for prototyping or debugging during development.